### DISCUSSION TITLE
Getting customized output results

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1233] [Questions & Problems]
Hello PK-Sim community,
I was trying to to show the effect of change of a parameter on the output result in the concentration-time profile curve. I wanted to overlay the results to show the changes in a single graph. How can I be able to do that? Is there any way to include multiple curves generated my changing a single parameter on a single graph?

For your reference I have attached an image. I want my output to be like this.

Regards,
Gautam Vijaywargi.
![Reference example](https://user-images.githubusercontent.com/107406825/174603091-2d6f5eb7-5b7c-413e-8bd1-760a83cf8603.JPG)

**Comment by Ricardo-DO** [comment]
Hi Gautam,

For the following examples, I used the midazolam template and changed the logP from ~3 to 0.

If you are simulating and individual, just click on the top in Compare results Individual Simulations

![image](https://user-images.githubusercontent.com/59565610/175136491-b5baaa5a-5db9-493c-9fc7-88a01a5482c2.png)

Then just drag and drop the simulations you want to compare

![image](https://user-images.githubusercontent.com/59565610/175136634-1cc8c361-c149-4fa5-b3db-872c500ee187.png)

![image](https://user-images.githubusercontent.com/59565610/175136723-b5311e68-e6e7-4fd2-a9ed-b01838abef6d.png)

You can do the same with populations.
When you have one of the Populations you would like to compare, click on Compare Results

![image](https://user-images.githubusercontent.com/59565610/175137087-ddb5af02-7a25-49b4-a6d1-8a3a6889353f.png)


Then select the populations you wish to compare and assign them colours and a Reference one, if you have it

![image](https://user-images.githubusercontent.com/59565610/175137592-4f7ebd74-c585-45b7-a090-e002a15f7f3e.png)

Then if you would like to show some area, you can click on Range 5% - 95%

![image](https://user-images.githubusercontent.com/59565610/175138023-6b293de3-fdfd-4cae-b852-90aeceb9abc8.png)

And that's it

![image](https://user-images.githubusercontent.com/59565610/175138245-5811d7a2-8732-4d78-a104-0575815c2661.png)

If you would like something exactly as the picture you're showing, you can export the data and create the plot in Prism.

I hope that helps.

Ricardo

**Reply by Gautamvj** [reply]
Thank you Ricardo for your elaborate reply. It has certainly helped me, but while performing it for population comparison I got stuck in the step as indicated in the image.
![Population comparison](https://user-images.githubusercontent.com/107406825/175220699-4f7d3209-0d12-438a-9a75-be3f264dfeba.JPG)
Here, as you can see the plasma concentration was not getting visible due to which I am unable to select the output. I tried multiple times but the same issue is getting repeated. Did I have missed something to add? Please help me with this.

Thank you.

**Reply by msevestre** [reply]
What output do you have selected in your simulation?

**Reply by StephanSchaller** [reply]
@Gautamvj, before selecting `compare results`, did you first simulate your population simulation?

**Reply by Gautamvj** [reply]
> What output do you have selected in your simulation?

I have selected venous plasma concentration

**Reply by Gautamvj** [reply]
> @Gautamvj, before selecting `compare results`, did you first simulate your population simulation?

Yes I have performed simulation before selecting the "compare results". Still the issue is persistent.

**Reply by Ricardo-DO** [reply]
> > @Gautamvj, before selecting `compare results`, did you first simulate your population simulation?
> 
> Yes I have performed simulation before selecting the "compare results". Still the issue is persistent.

Can you upload your PKSim file, so we can have a look? A simplified version should be enough just to see what's going on.

Cheers,
Ricardo

**Reply by Gautamvj** [reply]
> > > @Gautamvj, before selecting `compare results`, did you first simulate your population simulation?
> > 
> > 
> > Yes I have performed simulation before selecting the "compare results". Still the issue is persistent.
> 
> Can you upload your PKSim file, so we can have a look? A simplified version should be enough just to see what's going on.
> 
> Cheers, Ricardo

Hi Ricardo,
Here is a simplified version of my PKSim file in a ZIP format as the PKsim file format was not getting uploaded. Hope we can find a solution.
[Amphotericin-B.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/8973439/Amphotericin-B.zip)

Thank you.

**Reply by Ricardo-DO** [reply]
Hi Gautam,

I was checking your simulation and yes, you cannot select it for comparison.
I am not completely sure if this is because you have selected 2 outputs and PKSim cannot handle 2 at the same time.

The quick solution would be to right click, export to excel and use Prism or R to create plots like the one you are showing.
That one is a Prism plot.

Ricardo

**Reply by msevestre** [reply]
PKSim can absolutely handled more than two outputs at the same time.

**Reply by msevestre** [reply]
@Gautamvj You can only compare...the same outputs
On the one hand you have
![image](https://user-images.githubusercontent.com/1041237/176759922-bfe8d8b2-fcaa-4de5-a0b6-8a41c14029ae.png)

and on the other one you have
![image](https://user-images.githubusercontent.com/1041237/176759979-7d471643-3d1d-42ba-973c-27dfd57f4b8c.png)

Note that you have duplicated your compounds? The very first question I would ask is why?
Then I would suggest to check on the `building block` concepts in the online documentation 
https://docs.open-systems-pharmacology.org/open-systems-pharmacology-suite/modules-philsophy-building-blocks

If you want to experiment with different values of a parameter such as CL etc, change it in the simulation, compare and then update your BB with the better value

**Reply by Gautamvj** [reply]
> Hi Gautam,
> 
> I was checking your simulation and yes, you cannot select it for comparison. I am not completely sure if this is because you have selected 2 outputs and PKSim cannot handle 2 at the same time.
> 
> The quick solution would be to right click, export to excel and use Prism or R to create plots like the one you are showing. That one is a Prism plot.
> 
> Ricardo

Thank you Ricardo for giving your time. I appreciate your effort and surely will also try using Prism or R for building plots.

**Reply by Gautamvj** [reply]
> @Gautamvj You can only compare...the same outputs On the one hand you have ![image](https://user-images.githubusercontent.com/1041237/176759922-bfe8d8b2-fcaa-4de5-a0b6-8a41c14029ae.png)
> 
> and on the other one you have ![image](https://user-images.githubusercontent.com/1041237/176759979-7d471643-3d1d-42ba-973c-27dfd57f4b8c.png)
> 
> Note that you have duplicated your compounds? The very first question I would ask is why? Then I would suggest to check on the `building block` concepts in the online documentation https://docs.open-systems-pharmacology.org/open-systems-pharmacology-suite/modules-philsophy-building-blocks
> 
> If you want to experiment with different values of a parameter such as CL etc, change it in the simulation, compare and then update your BB with the better value

Hi Michael,
I have duplicated my compounds thinking that we will be needing them during comparison. One copy of compounds with initial values of a parameter and second set with optimized values and then we can plot them together to see the shift. 

Let me try your suggestion and come back to you.

Thank you.
